config/rbac/role.yaml

```diff
@@ -36,6 +36,19 @@ rules:
   - backups
   verbs:
   - create
+  - delete
```

Comment:
We need this updated in OADP CSV as well. Please update the corresponding OADP Operator PR: https://github.com/openshift/oadp-operator/pull/1588